swagger.yaml

```diff
@@ -2716,6 +2716,55 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/operators/bundles:
+    get:
+      tags:
+        - operators
+      summary: Get list of bundle names
+      description: Retrieves a list of bundle names.
+      operationId: V2GetBundles
+      responses:
+        "200":
+          description: Success
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/bundle'
+        "500":
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/error'
+
+  /v2/operators/bundles/{bundle_name}:
+    get:
+      tags:
+        - operators
+      summary: Get operator properties for a bundle
+      description: Retrieves an array of operator properties for the specified bundle.
+      operationId: V2GetBundleOperators
+      parameters:
+        - in: path
+          name: bundle_name
+          description: The name of the bundle.
+          required: true
+          type: string
+          enum: ['virtualization', 'openshiftai']
```

Comment:
Making this a enum means that every time that we add a new bundle we will need to modify the swagger.yaml file and regenerate all the code. I think it would be better to make this a plain string, maybe with a regular expression to disallow characters like `/`.

Reply:
I actually like that it's part of the API and not a plain string. it means that this field is validated at the API level.
Generating code is free